lib/thegarage/gitx/cli/base_command.rb

```diff
@@ -39,17 +37,21 @@ def current_branch
         end
 
         def aggregate_branch?(branch)
-          AGGREGATE_BRANCHES.include?(branch)
+          config.aggregate_branch?(branch)
         end
```

Comment:
Should this method just be removed entirely? If not, should there also be a method for abstracting config.reserved_branch? (see line 44 same file)?